addons/account/models/account_move.py

```diff
@@ -3676,6 +3676,13 @@ def _get_invoiced_qty_per_product(self):
                 qties[aml.product_id] -= qty
         return qties
 
+    def _has_residual_to_reconcile(self):
+        self.ensure_one()
+        residual_is_zero = self.company_currency_id.is_zero(self.amount_residual) \
+            and (not self.currency_id or self.currency_id.is_zero(self.amount_residual_currency))
```

Comment:
Is that check necessary ? If `self.amount_residual` is not zero, `self.amount_residual_currency` shouldn't be zero no?

Reply:
Maybe you are right, but I'd not remove extra check in stable

Reply:
It depends on the currency rounding. For instance `1 IDR = 0.00 EUR`